test/multifile/typealias/two-modules/library.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: true
```

Comment:
I've been abusing it lately to cover both intra- and inter-module issues.

Note that some of these tests were not specific to serialization, there were also SILGen issues. I'd rather keep them here as 'integration' tests for our multi-file support.

What do you think?